docs/resources/ingest_mapping.md

```diff
@@ -17,7 +17,7 @@ description: |-
 
 ### Required
 
-- `json_config` (String) Configuration of Ingest mapping in JSON format, the same one exported by Console.
+- `json` (String) Configuration of Ingest mapping in JSON format, the same one exported by Console.
```

Comment:
Run `make tfdocs_generate` to clean this up and regenerate documentation for other changes too